src/types/config.ts

```diff
@@ -1,12 +1,12 @@
 export type Config = {
-	apiBaseUrl: string;
-	figmaJson: boolean;
-	figmaUrl: string;
-	figmaToken: string;
-	figmaTokens: boolean;
-	figmaPage: string;
-	format: "css" | "js" | "json" | "sass" | "scss";
-	outputColorFormat: "hexadecimal" | "hsl" | "hsla" | "rgb" | "rgba";
+	apiBaseUrl: string; // Base URL from Figma API
+	figmaJson: boolean; // Export or not the original JSON from Figma API
+	figmaUrl: string; // URL from the project you need to extract the tokens
```

Comment:
```suggestion
	figmaUrl: string; // Figma's file URL param
```
Acho que até o nome do atributo poderia ser alterado. Tipo fileParam, ou fileId.

Reply:
4d54f77